NEWS.rst

```diff
@@ -2,6 +2,30 @@
 Release Notes
 =============
 
+----------------------------
+pkgcheck 0.10.11 (2022-05-26)
+----------------------------
+
+- EclassReservedName and EbuildReservedName: new check for usage of function or
+  variable names which are reserved for package manager (Arthur Zamarin, #378)
+
+- UrlCheck: skip verification of URLs with unknown protocol (Michał Górny, #384)
```

Comment:
```suggestion
- UrlCheck: skip verification of URLs with an unknown protocol. Such issues are already detected by BadUrl (Michał Górny, #384)
```

(I don't remember if it's called BadUrl, please fix that name, but you get the idea)

Reply:
Thank you (it was `DeadUrl`)